cmd/jujud/main.go

```diff
@@ -181,6 +181,7 @@ func jujuDMain(args []string, ctx *cmd.Context) (code int, err error) {
 	jujud.Register(unitAgent)
 
 	jujud.Register(NewUpgradeMongoCommand())
+	jujud.Register(agentcmd.NewCheckConnectionCommand(agentConf, agentcmd.ReallyConnect))
```

Comment:
The AgentConf that gets passed in is empty - it doesn't have a current conf until something calls `ReadConfig(agentName)` on it. Or are you concerned about the conf being shared between the different commands? I can change that, anyway.